interp/interp.go

```diff
@@ -58,10 +59,11 @@ type Exports map[string]map[string]reflect.Value
 
 // opt stores interpreter options
 type opt struct {
-	astDot bool   // display AST graph (debug)
-	cfgDot bool   // display CFG graph (debug)
-	noRun  bool   // compile, but do not run
-	goPath string // custom GOPATH
+	astDot  bool   // display AST graph (debug)
+	cfgDot  bool   // display CFG graph (debug)
+	noRun   bool   // compile, but do not run
+	goPath  string // custom GOPATH
+	context build.Context
```

Comment:
Please add a comment for this field like: `// To handle build constraints, and skip unwanted source files`